conf/test.config

```diff
@@ -92,18 +96,29 @@ profiles {
     umi_quiaseq {
         params.genome              = 'smallGRCh38'
```

Comment:
Wasn't this genome removed?

Reply:
this whole thing is the old test form sarek 2.7.2. I haven't updated the csv yet to use the reads in test-data/modules. I should do that though. Almost forgot, thanks for reminding me :)